src/processTranslation.tsx

```diff
@@ -31,6 +32,7 @@ interface ProcessTranslationParams {
 
 export const processTranslation = ({
   locale,
+  numberLocale = locale,
```

Comment:
Do we need adding new parameter when there is no usage of `locale` after this change?